src/views/users/const.js

```diff
@@ -100,3 +100,13 @@ export const userJSONSelectMeta = (vm, withoutOrgRole = false) => {
     }
   }
 }
+export const MFALevel = {
+  allUsers: 3,
+  onlyAdminUsers: 4
+}
+
+export const MFASystemSetting = {
+  notEnable: 0,
+  allUsers: 1,
+  onlyAdminUsers: 2
+}
```

Comment:
The provided code has two changes:

1. The function `MFALevel` definition has been simplified to reduce the number of exported constants it contains.

2. Some unnecessary variables have been removed.

These modifications should improve efficiency, while also maintaining clarity in the code structure if needed.